new_component_todo-item.md

```diff
@@ -47,23 +49,23 @@ Now we need to pass a string, which is the item's title, where we use the compon
 </ul>
 ```
 
-The `itemTitle` here in square brackets is the same as declared as the component's `@Input`.
+O `itemTitle` entre colchetes é o mesmo que foi declarado como componente no `@Input`.
 
-We use property binding on an element we created ourselves! And now we can actually see and understand that property binding binds to actual property of the component. 
+Nós usamos a propriedade Binding em um elemento que nós mesmo criamos! E agora podemos realmente ver  e entender que a vinculação da propriedade Binding se liga a propriedade atual do componente.
 
-## Passing the whole item
-We will refactor our code a bit so we can easily implement more functionality in the `todo-item` component, for example editing and removing the item. Instead of passing just the title to the component, we will pass the whole item, and let the component extract the title where needed.
+## Passando o item inteiro
+Agora iremos refatorar um pouco nosso código para que possamos facilmente implementar mais funcionalidades no componente `todo-item`, por exemplo editando e removendo o item. Em vez de passar apenas o título ao componente, nós poderiamos passar todo o item, e deixamos o componente extrair o título quando necessário.
```

Comment:
Pls, check the comma and connection between sentences.

Reply:
Thanks, corrected #063c991.